studybuddies/src/app/tutor/class-list-tutor/class-list-tutor.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ClassListTutorComponent } from './class-list-tutor.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7155399)
```suggestion
import { ClassListTutorComponent } from "./class-list-tutor.component";
```